jwst/outlier_detection/outlier_detection_tso.py

```diff
@@ -66,6 +70,9 @@ def do_detection(self):
         log.info("Flagging outliers")
         flag_cr(self.input_models, median_model, **self.outlierpars)
 
+        # Make sure all arrays have matching NaNs and DQ flags
+        for input_model in self.input_models:
```

Comment:
`self.input_models` is a `CubeModel` here. The iteration appears to produce the result of "keys" (so a `input_model` will be a string). Looking at `match_nans_and_flags` this should produce a warning?

Reply:
Good catch, thanks!  In this case, it didn't throw a warning - it just quietly did nothing because it didn't have the attributes it was looking for.  Fixing this case now.

Reply:
This should be fixed now.